src/testdir/test_vim9_cmd.vim

```diff
@@ -2036,6 +2036,14 @@ def Test_no_space_after_command()
   v9.CheckDefExecAndScriptFailure(lines, 'E486:', 1)
 enddef
 
+def Test_lambda_crash()
+  # This used to crash Vim
+  var lines =<< trim END
+    vim9 () => super      => {
+  END
+  v9.CheckSourceFailure(lines, 'E1356: "super" must be followed by a dot', 1)
```

Comment:
```suggestion
v9.CheckScriptFailureList(lines, ["E1356:", "E1405:"])
```
This tests  for both errors, including the one that got the SEGV.

Reply:
Thanks, I fix it